UnitTests/GitCommands.Tests/Helpers/PathUtilTest.cs

```diff
@@ -347,9 +333,36 @@ public void DeleteWithExtremePrejudice_should_return_true_if_delete_successful()
         [TestCase("https://github.com/gitextensions/gitextensions", true)]
         [TestCase("https://github.com/gitextensions/gitextensions.git", true)]
         [TestCase("github.com/gitextensions/gitextensions.git", true)]
+        [TestCase("HTTPS://MYPRIVATEGITHUB.COM:8080/LOUDREPO.GIT", true)]
+        [TestCase("git://myurl/myrepo.git", true)]
+        [TestCase("github.com/gitextensions", false)]
+        [TestCase("github.com/gitextensions/gitextensions/pull/9018", false)]
+        [TestCase("http://", false)]
+        [TestCase("HTTPS://www", true)]
+        [TestCase("git://", true)]
+        [TestCase("file:", false)]
+        [TestCase("SSH:", true)]
+        [TestCase("SSH", false)]
+        [TestCase("https://myhost:12368/", true)]
         public void CanBeGitURL(string url, bool expected)
```

Comment:
There are no tests for the second tests

```c#
                   || url.EndsWith(".git", StringComparison.CurrentCultureIgnoreCase)
                   || GitModule.IsValidGitWorkingDir(url);
```

Reply:
There are some tests for paths/URLs that end with .git but I guess there are no tests for IsValidGitWorkingDir since it needs a specific directory structure.